src/app/layouts/header/import-popup/import-popup.component.html

```diff
@@ -1,14 +1,30 @@
 <app-popup
-    title="Import a Theme"
+    title="Theme Import"
     #popup
 >
     <div class="text">
```

Comment:
I think that this window should look like here
https://github.com/DevExpress/ThemeBuilder/pull/77/files#diff-4304e88873258eb9d1d67c7da49eae3dR7